### ISSUE TITLE
Update: Select PLA SSM/ASM [WIP, see CMO discord]

### ISSUE THREAD

**Issue by HunterDNiu** [PygmalionOfCyprus/cmo-db-requests#4856]
[PLA SSM ASM.xlsx](https://github.com/user-attachments/files/16440093/PLA.SSM.ASM.xlsx)

**Comment by LuigiP57** [comment]
Is your Discord the same nickname as you are on Github?

**Comment by LuigiP57** [comment]
If the KF-98A has ARH/PRH (my ticket is IIR+PRH), then it should be able to take on Surface/S targets？I see you gave the KD-88C G/S. And, in addition to the Submunition. HE/SAP warheads, they are also replaceable. I don't think it's just a variant of GB-6 dispenser. Note that in some photos, they are simultaneously equipped by the PLAAF.

**Comment by HunterDNiu** [comment]
> If the KF-98A has ARH, then it should be able to take on Surface/S targets？I see you gave the KD-88C G/S. And, in addition to the Submunition. HE/SAP warheads, they are also replaceable. I don't think it's just a variant of GB-6 dispenser. Note that in some photos, they are simultaneously equipped by the PLAAF.

Yeah could you please make a google doc stuff cause I don't really know how.

**Comment by LuigiP57** [comment]
> Yeah could you please make a google doc stuff cause I don't really know how.

I think this is OK for now, if it can be edited online, I don't know if it will lead to some "mess".

**Comment by HunterDNiu** [comment]
> > Yeah could you please make a google doc stuff cause I don't really know how.
> 
> I think this is OK for now, if it can be edited online, I don't know if it will lead to some "mess".

![Declaration_of_Independence_(1819),_by_John_Trumbull](https://github.com/user-attachments/assets/b8b75c98-d7fa-4ece-ba64-a32a3710d727)
:)

**Comment by LuigiP57** [comment]
And I have to say, database update of v506 & v507/a is taking so long. Maybe a good portion of your requests have already been completed, we just don't know it.

**Comment by yU2187sie** [comment]
Yes, the database update on PLAAF has long been completed, but due to the update time we are unable to obtain the 506,507 /a database. There are enough issues on GitHub about the PLAAF's various new munitions that you can try to look at the past records, and many of them are of good quality.
Hopefully this will save you a little more energy to put into other work:)
